python-squid/src/main/java/org/sonar/python/tree/BaseTreeVisitor.java

```diff
@@ -260,4 +263,22 @@ public void visitWithItem(PyWithItemTree pyWithItemTree) {
     scan(pyWithItemTree.test());
     scan(pyWithItemTree.expression());
   }
+
+  @Override
+  public void visitQualifiedExpression(PyQualifiedExpressionTree pyQualifiedExpressionTree) {
+    scan(pyQualifiedExpressionTree.qualifier());
+    scan(pyQualifiedExpressionTree.name());
+  }
+
+  @Override
+  public void visitCallExpression(PyCallExpressionTree pyCallExpressionTree) {
+    scan(pyCallExpressionTree.callee());
+    scan(pyCallExpressionTree.arguments());
+  }
+
+  @Override
+  public void visitArgument(PyArgumentTree pyArgumentTree) {
+    scan(pyArgumentTree.keywordArgument());
+    scan(pyArgumentTree.expression());
```

Comment:
I see that the methods of `BaseTreeVisitor` never scan tokens. I'm not sure it's a good thing. Will we be able to rewrite CPD integration for example?